### PR TITLE
[WIP][No-7][Training]create-blog[REVIEW]

### PR DESCRIPTION
**Pull request by nguyentienthinh**
Task name: Create blog.
Content need review: Create.
Checked with phpcs PSR-2, phpmd: success.
Link task: https://docs.google.com/spreadsheets/d/1aWRDQ4TsW6BgTjm7iLPnI7R9VRPeZF0Fl6lEq-wfpa8/edit?folder=1KnNubuBhStnVllQ-62F3SW1mG3x9DT-G#gid=2142888040

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -31,12 +31,14 @@
...
         Route::get('/users', 'AdminController@users')->name('admin.users');
-        Route::get('/blogs', 'AdminController@blogs')->name('admin.blogs');
+        // Route::get('/blogs', 'AdminController@blogs')->name('admin.blogs');
     });
 
     //Route User
     Route::namespace('User')->group(function () {
         // Controllers Within The "App\Http\Controllers\User" Namespace
-        Route::get('/', 'BlogController@index')->name('blog.index');
+        // Route::get('/', 'BlogController@index')->name('blog.index');
```
Comment (phantuan1803) [inline routes/web.php]: Code not use, no comment

**Review by phantuan1803 — COMMENTED** [review]
Please fix

File: routes/web.php (hunk 1 above)
Comment (nguyentienthinh) [reply routes/web.php]: ok, thanks

**Comment by phantuan1803** [comment]
LGTM

File: resources/views/blog/index.blade.php [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <form action="{{route('blog.store')}}" name="form-create" method="POST" enctype="multipart/form-data">
+        @csrf
+        <div class="modal-body">
+          <div class="form-group">
+              <label for="blog-title" class="col-form-label">Title:</label>
+              <input type="text" class="form-control" id="blog-title" name="title">
+            </div>
```
Comment (truonggiang1836) [inline resources/views/blog/index.blade.php]: Please fix indent text and apply for same line

Comment (nguyentienthinh) [reply resources/views/blog/index.blade.php]: I fixed, please help me review this task

**Review by truonggiang1836 — APPROVED** [review]
LGTM

**Pull request closed without merging**